react-native-libraries.json

```diff
@@ -9482,5 +9482,16 @@
     "ios": true,
     "android": true,
     "web": true
+  },
+  {
+    "githubUrl": "https://github.com/mutualmobile/react-native-barricade",
+    "npmPkg": "@mutualmobile/react-native-barricade",
+    "nameOverride": "react-native-barricade",
```

Comment:
Hello @prajnab-mm, thank you for adding new entry to the Directory! 🙏 

```suggestion
```

Let's remove the name overwrite, since it can confuse people, since the package is scoped. Besides that the addition LGTM! 👍